### ISSUE TITLE
Bot is not adding comment in the PR

### ISSUE THREAD

**Issue by kspawan** [anc95/ChatGPT-CodeReview#141]
I tried using the same configuration but it is not adding comments to the PR. I thought of some access issue with GITHUB_TOKEN but I tried using personal access token too, nothing worked out.

Is there anything from your side or am i doing something wrong ?

**Comment by anc95** [comment]
have you set OPEN_API_KEY in the variables?

**Comment by kavinsharma** [comment]
> have you set OPEN_API_KEY in the variables?

yes, i have added OPENAI_API_KEY in the env file and deployed on lambda, but when making a pull request, and the chatAPI.sendMessage function is being called in the lambda.index file, then aws lambda is timing out, not providing result.

it is working on local as it is taking the function then from the node modules

**Comment by anc95** [comment]
try to add up the timeout value for aws lambda
